setup.md

```diff
@@ -0,0 +1,322 @@
+# Main setup instructions for the Oxford Computational Biochemistry course
+
+The Oxford Computational Biochemistry course contains 4 self-contained
+tutorials which each have different software installation requirements. These
+instructions can be found in the respective tutorial's folder. Here we
+provide some basic instructions on the base setup requirements common to these
+tutorials and how to obtain a local copy of the materials.
+
+In order to ensure that you have everything ready in advance of the workshop,
+we ask that you follow these setup instructions before attending each tutorial.
+Q&A sessions will be setup on the two days preceeding the course to help you
+with any issues you might encounter during the setup process.
+
+
+## 1. [Anaconda]
+
+All tutorials in this course rely heavily on the [Anaconda] package manager.
+This cross platform tool allows you to install other software packages in an
+easy and controllable manner. We require all attendees to install this prior
+to starting any of the tutorials. Please see the follow the [Anaconda install instructions]
+to get started.
+
+The [Anaconda] package manager, is named [conda]. Throughout the tutorials you
+will see several invocations of [conda], including to create [environments] and
+install software. If you are unfamiliar with this tool, it is heavily
+recommened that you go through the [getting started guide].
+
+
+## 2. Using the terminal
+
+Throughout the tutorials we will be interacting with various software through
+the use of a terminal interface. Terminals are a powerful means by which to
+navigate through files and quickly execute various programs.
+
+On linux/macOS this is generally done though a bash/zsh shell.
+
+- In linux this can be accessed by searching for terminal in the startup menu.
+  See below for an example in the ubuntu gnome shell.
+
+![Ubuntu shell](./images/main-setup/ubuntu-terminal.png)
+
+- On macOS this is done by searching for terminal in the Launchpad (see the
+  [following for more details][apple terminal support]).
+
+(Todo: Insert picture of opening terminal in macOS here)
+
+On Windows (unless using WSL in which case please follow the linux instructions),
+this will be done via the Anaconda prompt. The Anaconda prompt can be opened by
+searching for it in the startup menu after having installed Anaconda (see the
+step above).
+
+![Anaconda Prompt](./images/main-setup/anaconda-terminal.png)
```

Comment:
@philbiggin can you confirm that this is the prompt that was used to test out the docking & homology modelling tutorials under windows?

Reply:
Yes - confirmed that is what I did.